alternator/executor.cc

```diff
@@ -2216,6 +2215,7 @@ future<executor::request_return_type> executor::batch_write_item(client_state& c
                     co_return api_error::validation("Provided list of item keys contains duplicates");
                 }
                 used_keys.insert(std::move(mut_key));
+                batch_size++;
```

Comment:
I didn't understand the commit message. You used the word "inaccurate" and "incorrect", but what was actually incorrect? I was worried in my review that this code actually counts the number of tables, not the number of items. Is this it? If so, please say so and don't say "inaccurate". If it's not it, please explain what was the inaccuracy. Thanks.

Reply:
I'll change the commit message to make what was wrong clearer.